controllers/disruption_controller.go

```diff
@@ -493,8 +495,14 @@ func (r *DisruptionReconciler) startInjection(ctx context.Context, instance *cha
 				continue
 			}
 
-			if chaosv1beta1.ShouldSkipNodeFailureInjection(disKind, instance, injection) {
-				r.log.Debugw("skipping over injection, seems to be a re-injected node failure", "targetName", targetName, "injectionStatus", injection)
+			injection := injections.GetInjectionWithDisruptionKind(disKind)
+
+			if injection == nil {
+				return fmt.Errorf("the injection status from the target injections with this %s kind of disruption does not exist", disKind)
```

Comment:
when would we hit this condition?

Reply:
IMO I will said not, but if it's the case I prefered to have an explicit error rather an invalid memory access error which is more hard to debug